### DISCUSSION TITLE
Fall Detection Model Training

### DISCUSSION THREAD

**Discussion by jjsenay** [ambianic/ambianic-edge#411] [General]
I repaired my car in my driveway. Lots of Falls detected. Some were false. Use for training the AI model 20211214-153553.415494.zip

Enjoy!!
[20211214-153553.415494.zip](https://github.com/ambianic/ambianic-edge/files/7721570/20211214-153553.415494.zip)
!

**Comment by ivelin** [comment]
This is helpful data for future training rounds.
 
The current model performance is optimal when the distance is 10-20 feet. 
More on Practical Workaround and Troubleshooting:
https://blog.ambianic.ai/2021/09/02/movenet-vs-posenet-person-fall-detection.html#practical-workarounds-and-troubleshooting
